packages/sdk/src/resolve/resolve-accounts.js

```diff
@@ -85,6 +107,18 @@ async function collectAccounts(ix, accounts, last, depth = 3) {
 
 export async function resolveAccounts(ix) {
   if (isTransaction(ix)) {
+    if (!Array.isArray(ix.payer)) {
+      console.warn(
+        `
+        %cFCL Warning
+        ============================
+        "ix.payer" must be an array. Support for ix.payer as a singular is deprecated,
+        see changelog for more info: https://docs.onflow.org/fcl/
```

Comment:
We should point this link to a CHANGELOG entry in sdk

Reply:
I'm having trouble with this link. Going to merge without the link for now and we can revisit this before we release 1.0.